File: cfgs/java_cfgs_foo/STRING_CONTAINING_FIRST_LETTER_EVERY_WORD_GIVEN_STRING_SPACES.java
Nodes
1;special;;;
2;statement;String result = "";14;
3;statement;boolean v = true;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < fun_1();17;
6;conditional;fun_1(foo_2) == ' ';19;
7;statement;v = true;20;
8;conditional;(fun_1(foo_2) != ' ') && (foo_3 == true);22;
9;statement;result += fun_1(foo_2);23;
10;statement;v = false;24;
11;statement;i++;18;
12;statement;return foo_1;27;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;12;
5;7;8;
6;11;
7;9;11;
8;10;
9;11;
10;5;
11;13;
12;

File: cfgs/pythoncfgs/STRING_CONTAINING_FIRST_LETTER_EVERY_WORD_GIVEN_STRING_SPACES.py
Nodes
1;statement;foo_1 = '';8;
2;statement;foo_1 = true;9;
3;conditional;fun_2(fun_2(foo_1));10;
4;statement;return foo_1;16;
5;conditional;foo_1[foo_2] == ' ';11;
6;conditional;foo_1[foo_2] != ' ' and foo_3 == true;13;
7;statement;foo_1 += foo_2[foo_3];14;
8;statement;foo_1 = false;15;
9;statement;foo_1 = true;12;
10;exit;;
Edges
0;2;
1;3;
2;4;5;
3;10;
4;6;9;
5;3;7;
6;8;
7;3;
8;3;
9;